sql/src/main/java/io/crate/execution/dsl/package-info.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+/**
+ * <p>
+ *  This package contains classes for the execution domain-specific language (DSL).
```

Comment:
Does it really qualify as a DSL? A complete DSL is self-sufficient but this one you still have to use with other classes.

Reply:
What else what you use? DSL is a farily generic term in my book =)

Reply:
A DSL is a complete language for a domain. I don't think that is true here because the classes inside DSL don't make up a complete language for writing executions with crate. 

IMHO API would fit.

Reply:
`API` is imho too generic. I pushed another fixup with a bit more introduction. That should clarify why I think DSL fits. (The DSL here is for the *generic* execution framework, that is, the parts we use to execute all query based statements. The ExecutionPhasesTask could probably be seen as entry point). If viewed like that, this does include at least all "entry-point" classes which are used to describe the execution. 
(Symbols are more generic than that, which is why they shouldn't go into this package).
(LogicalPlan is just a means to create (among others) the executionPlan - but the execution layer I'm referring here too has no knowledge of that, and shouldn't have)

Reply:
What do you mean by Symbols? Symbols in queries?

I wouldn't call it DSL but I can see how the term can be used to describe an execution API. What matters is the restructuring and I think that's great.